### PR TITLE
[Custom Amounts M1] Prepare for release

### LINKED ISSUE #10883: [Custom Amounts M1] Remove Feature Flag to enable functionality in Release
Remove also the code we won't need any more: set an unique fee to the order.

### PR DESCRIPTION
**Pull request by toupper**
Closes: #10883

## Description

With this PR we prepare the first milestone of the Custom Amounts project for release by:

- Removing the feature flag and related code
- Removing the legacy code related to adding a single fee to the order
- Updating UI fees references to show "Custom amounts" instead
- Updating Unit Tests
- Updating UI Tests (In progress, shouldn't affect this PR review)

## Testing instructions

Please make sure that the custom amounts functionality works on Release. Also, see that we don't have references to "fees" neither in order creation or order details:

https://github.com/woocommerce/woocommerce-ios/assets/1864060/66343775-686b-40fb-a4f8-2cebd6d94245

---
- [X] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomAmounts/OrderCustomAmountsSection.swift [hunk 1]
```diff
@@ -62,7 +62,7 @@ private extension OrderCustomAmountsSection {
         static let addCustomAmount = NSLocalizedString("Add Custom Amount",
                                                    comment: "Title text of the button that allows to add a custom amount when creating or editing an order")
         static let customAmounts = NSLocalizedString("orderForm.customAmounts",
-                                                     value: "Custom Amounts",
+                                                     value: "Custom amounts",
```
Comment (joshheald) [inline WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomAmounts/OrderCustomAmountsSection.swift]: This should be title case on iOS: pecCkj-eD-p2

```suggestion
                                                     value: "Custom Amounts",
```

It's also nicer for it to be consistent with the row above
order-details-custom-amounts-capitalisation

File: WooCommerce/Classes/ViewRelated/ReusableViews/LedgerTableViewCell.swift [hunk 2]
```diff
@@ -106,7 +106,7 @@ private extension LedgerTableViewCell {
     enum Titles {
         static let subtotalLabel = NSLocalizedString("Product Total",
                                                      comment: "Product Total label for payment view")
-        static let feesLabel = NSLocalizedString("Fees",
+        static let feesLabel = NSLocalizedString("Custom amounts",
```
Comment (joshheald) [inline WooCommerce/Classes/ViewRelated/ReusableViews/LedgerTableViewCell.swift]:
```suggestion
        static let feesLabel = NSLocalizedString("Custom Amounts",
```

Nicer again for it to match the row above
custom amounts capitalisation

File: WooCommerce/WooCommerceTests/ViewRelated/LedgerTableViewCellTests.swift [hunk 3]
```diff
@@ -96,8 +96,8 @@ private extension LedgerTableViewCellTests {
     enum Titles {
         static let subtotalLabel = NSLocalizedString("Product Total",
                                                      comment: "Product Total label for payment view")
-        static let feesLabel = NSLocalizedString("Fees",
-                                                     comment: "Fees label for payment view")
+        static let customAmountsLabel = NSLocalizedString("Custom amounts",
+                                                     comment: "Custom amounts label for payment view")
```
Comment (joshheald) [inline WooCommerce/WooCommerceTests/ViewRelated/LedgerTableViewCellTests.swift]:
```suggestion
        static let customAmountsLabel = NSLocalizedString("Custom Amounts",
                                                     comment: "Custom amounts label for payment view")
```

**Review by joshheald — APPROVED** [review]
Works as expected 🚢 

A couple of typography correction suggestions – can be done separately, but nice to do right from the beginning...

**Comment by toupper** [comment]
Thanks for the review and the typos suggestion @joshheald ! I committed them, it's good to be consistent all along

**Pull request merged**